qa/tasks/admin_socket.py

```diff
@@ -152,8 +152,14 @@ def _run_tests(ctx, client, tests):
 
             test_path = None
             if 'test' in config:
+                # hack: the git_url is always ceph-ci or ceph
+                git_url = teuth_config.get_ceph_git_url()
```

Comment:
Could just pass the full url (minus the file) via a new variable (e.g. repo_url_base = git_url + "?p=" + repo_name + ";a=blob_plan;hb=" + branch + ";f=") since logs of rbd workunits need fixes like this as well.